simpledotorg/simple-server
# Exclude inactive facilities

**Story card:** [sc-7482](https://app.shortcut.com/simpledotorg/story/7482/exclude-inactive-facilities-from-csv-download)

## Because

Exclude inactive facilities from CSV download
- In the "district sheet" change “Facilities implementing IHCI” from total facility count to exclude inactive facilities. 
- In the "facility sheet" exclude inactive facilities

## This addresses

Add a scope `active` to the facility model which takes a `month_date` and returns only the active facilities in that `month_date`

## Test instructions

- Make sure to enable `monthly_district_report` in Flipper
- Download "Monthly district report" on the Reports page.  You should now only be able to see the active facilities in the "district sheet" and the "facility sheet"
